www/directives/vehicle/vehicle-directive.html

```diff
@@ -22,12 +22,12 @@
   <div class="row item-text-wrap text-left" style="padding-top: 10px">
     <!-- Last stop -->
     <div class="col col-50 item-text-wrap" style="text-align: left;">
-      <i class="icon ion-ios-location" aria-label="last location of bus"></i>
+      <i class="icon ion-ios-location" aria-hidden="true"></i>
```

Comment:
I'd argue we should keep this one read out loud.  All the other properties of vehicles are pretty self-explanatory ("To North Amherst," "6 riders," "4 minutes late", "on time), but simply reading the name of a stop might cause confusion.  

An example confusing readout here could be "Puffers Pond," which could mean the destination of the bus, it could be the last stop, or perhaps even the next stop.